amun/configuration.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-# Amun
+# thoth-user-api
```

Comment:
Should it be 'amun-api'?

Reply:
Fixed, PTAL. The openapi v3 compliant service with these changes is already running in the test environment.